packages/react/ds/src/accordion/accordion-item.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react';
+import { cn } from '../cn.js';
+import { Icon } from '../icon/icon.js';
+
+type Props = {
```

Comment:
Can we have a prop to let the user choose to make the icon appear on the left or right?

Reply:
fixed